opentreemap/importer/templates/importer/partials/merge_species.html

```diff
@@ -21,7 +21,7 @@
                             <input type="radio"
                                    id="{{ value.id }}"
                                    name="{{ field.id }}"
-                                   value="{{ value.value }}"
+                                   value="{{ value.value|unlocalize }}"
```

Comment:
If I am looking in the right place, this `value` is not a number

<img width="1391" alt="screen shot 2015-10-21 at 7 56 09 am" src="https://cloud.githubusercontent.com/assets/17363/10640420/69d9bc6e-77c9-11e5-84e1-20d1d6713705.png">